.babelrc

```diff
@@ -1,7 +1,10 @@
 {
   "presets": [
     "react",
-    ["es2015", { "loose": true }],
+    ["env", { "targets": {"node": "current"}, "loose": true }],
     "stage-2"
+  ],
+  "plugins": [
+    "add-module-exports"
```

Comment:
It appears that just using the `react` and `env` presets will suffice. We can drop the `stage-2` preset, and I don't think we need the `add-module-exports` plugin.

Reply:
There are two transforms used that are not part of `babel-preset-env`, I've added them individually as that is more self documenting (and IMHO safer) than the stage presets. The one for object rest/spread can be dropped with Babel7.

 add-module-exports is necessary as the tests don't run through rollup.